tests/TestGui/simulation_flow.tcl

```diff
@@ -0,0 +1,43 @@
+#Copyright 2021 The Foedag team
+
+#GPL License
+
+#Copyright (c) 2021 The Open-Source FPGA Foundation
+
+#This program is free software: you can redistribute it and/or modify
+#it under the terms of the GNU General Public License as published by
+#the Free Software Foundation, either version 3 of the License, or
+#(at your option) any later version.
+
+#This program is distributed in the hope that it will be useful,
+#but WITHOUT ANY WARRANTY; without even the implied warranty of
+#MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#GNU General Public License for more details.
+
+#You should have received a copy of the GNU General Public License
+#along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+create_design simulation_flow_test
```

Comment:
How is this one passing without an arch or target device selection?

Reply:
https://github.com/os-fpga/FOEDAG/pull/567

Reply:
Yes, because it is dummy compiler. Fake compiler that doesn't have any checks.